src/transformState.js

```diff
@@ -5,7 +5,39 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(action => {
+    const { type } = action;
+
+    switch (type) {
+      case 'addProperties':
```

Comment:
Move all cases to the object. Example of usage: `case Actiontype.AddProperties:`